packages/venia-ui/lib/components/Checkout/shippingForm.js

```diff
@@ -35,16 +35,21 @@ const ShippingForm = props => {
 
     const classes = mergeClasses(defaultClasses, props.classes);
 
+    const formHeading = 'Shipping Information';
+    const shippingMethodLabel = 'ShippingMethod';
```

Comment:
Looks like this string should be `Shipping Method`

Reply:
Oops! Fixed.